.github/workflows/stale.yml

```
@@ -14,10 +14,11 @@ jobs:
     steps:
       - uses: actions/stale@v6
         with:
```

```suggestion
        with:
          repo-token: "${{ secrets.BOT_TOKEN }}"
```

This change lets the stale issue comments be posted under the account of our adapter-hub-bert bot. Not really required, but we do this throughout all repos (cf. e.g. https://github.com/adapter-hub/Hub/blob/master/.github/workflows/label.yml)